src/docdb/tree/DocDBDatabaseTreeItemBase.ts

```diff
@@ -64,58 +65,53 @@ export abstract class DocDBDatabaseTreeItemBase extends DocDBTreeItemBase<Collec
 
     // Create a DB collection
     public async createChild(_node: IAzureNode, showCreatingNode: (label: string) => void): Promise<IAzureTreeItem> {
-        const collectionName = await vscode.window.showInputBox({
+        const collectionName = await ext.ui.showInputBox({
             placeHolder: `Enter an id for your ${this.childTypeLabel}`,
             ignoreFocusOut: true,
             validateInput: DocDBDatabaseTreeItemBase.validateCollectionName
         });
 
-        if (collectionName) {
-            let partitionKey: string | undefined = await vscode.window.showInputBox({
-                prompt: 'Partition Key',
-                ignoreFocusOut: true,
-                validateInput: DocDBDatabaseTreeItemBase.validatePartitionKey
-            });
+        let collectionDef: Collection = {
+            id: collectionName
+        };
 
-            if (partitionKey) {
-                if (partitionKey[0] !== '/') {
-                    partitionKey = '/' + partitionKey;
-                }
-                const throughput: number = Number(await vscode.window.showInputBox({
-                    value: minThroughput.toString(),
-                    ignoreFocusOut: true,
-                    prompt: `Initial throughput capacity, between ${minThroughput} and ${maxThroughput}`,
-                    validateInput: DocDBDatabaseTreeItemBase.validateThroughput
-                }));
-
-                if (throughput) {
-                    const options = { offerThroughput: throughput };
-                    const collectionDef = {
-                        id: collectionName,
-                        partitionKey: {
-                            paths: [partitionKey],
-                            kind: DocumentBase.PartitionKind.Hash
-                        }
-                    };
-
-                    showCreatingNode(collectionName);
-                    const client = this.getDocumentClient();
-                    const collection: CollectionMeta = await new Promise<CollectionMeta>((resolve, reject) => {
-                        client.createCollection(this.link, collectionDef, options, (err, result) => {
-                            err ? reject(err) : resolve(result);
-                        });
-                    });
-
-                    return this.initChild(collection);
-                }
-            }
+        let partitionKey: string | undefined = await ext.ui.showInputBox({
+            prompt: 'Enter the partition key for the collection, or leave blank in case of fixed size.',
```

Comment:
or leave blank to create a fixed-size collection

Reply:
The line might be too long.